postgresql-ballerina/client.bal

```diff
@@ -126,103 +125,6 @@ public client class Client {
     }
 }
 
-# Provides a set of configurations for the PostgreSQL client to be passed internally within the module.
```

Comment:
Why are these removed?

Reply:
Those are removed from this as these are moved to [record_type.bal](https://github.com/ballerina-platform/module-ballerinax-postgresql/pull/7/files#diff-00377e4abd9f8117b7b23ef8d6f98a88079e8394b7d512954f2f76212607bf97R21)